src/main/drivers/dshot_bitbang_impl.h

```diff
@@ -123,7 +137,11 @@ typedef struct bbPort_s {
     uint16_t dmaSource;
 
     dmaResource_t *dmaResource; // DMA resource for this port & timer channel
+#ifdef AT32F435
+    uint32_t dmaMux;
+#else
     uint32_t dmaChannel;        // DMA channel or peripheral request
```

Comment:
Perhaps rename this to simple dmaChannelOrMux or simply use dmaChannel

Reply:
changed to dmaChannel